frontend/common/stores/project-store.js

```diff
@@ -162,6 +169,24 @@ const store = Object.assign({}, BaseStore, {
     })
   },
   getEnvs: () => store.model && store.model.environments,
+  getMaxFeaturesAllowed: () => {
+    return store.model && store.model.max_features_allowed
+  },
+  getMaxSegmentOverridesAllowed: () => {
```

Comment:
- Can we update responses.ts to include these new types?
- It looks like this is quite a big breaking change, right now e2e is failing because all of these fields are undefined. 
- Should these be controlled via feature flag or assume undefined means Number.MAX_SAFE_INTEGER?

Reply:
types and Number.MAX_SAFE_INTEGER added.

Reply:
These should never be undefined. It's perhaps just an issue that the store is getting updated from multiple places? I don't think it's wise to default to `Number.MAX_SAFE_INTEGER` when that is likely not the actual value. Can we please determine _why_ we're seeing undefined values and look at how we resolve them?

Reply:
Right, as mentioned in a comment above these values should never be undefined, as the API is currently designed. Does it hurt if we have those Max Int values as a fallback? The issue was the logic to disable the creation of new segment overrides, that logic was corrected as part of the prior commit.

Reply:
Marked as resolved as per the comments above